test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14227,15 +14227,18 @@ def create_eatcpu_job(self, duration=None, mom=None):
             self.du = DshUtils()
         script_dir = os.path.dirname(os.path.dirname(__file__))
         script_path = os.path.join(script_dir, 'utils', 'jobs', 'eatcpu.py')
+        if not self.du.is_localhost(mom):
+            d = pwd.getpwnam(self.username).pw_dir
+            ret = self.du.run_copy(hosts=mom, src=script_path, dest=d)
+            if ret is None or ret['rc'] != 0:
+                raise AssertionError("Failed to copy file %s"
```

Comment:
Please include mom name in error message like `"Failed to copy file %s to %s" % (script_path, mom)`